hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -157,15 +159,48 @@ def setUp(self):
 
         s.set_elements(elements)
         s.add_lines(xray_lines)
+        s.axes_manager[0].scale = 0.5
+        s.axes_manager[1].scale = 0.5
         self.signal = s
 
     def test_quant_lorimer(self):
         s = self.signal
+        method = 'CL'
         kfactors = [1, 2.0009344042484134]
+        composition_units = 'weight'
         intensities = s.get_lines_intensity()
-        res = s.quantification(intensities, kfactors)
+        res = s.quantification(intensities, method, kfactors, composition_units)
+        nt.assert_true(np.allclose(res[0].data, np.array([[22.70779, 22.70779],
+                    [22.70779, 22.70779]]), atol=1e-3))
+
+    def test_quant_zeta(self):
+        s = self.signal
+        method = 'zeta'
+        compositions_units = 'weight'
+        factors = [20, 50]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors, compositions_units)
+        nt.assert_true(np.allclose(res[1].data, np.array(
+                [[  16.216216216,   16.216216216],
+                [  16.216216216,   16.216216216]]), atol=1e-3))
+        nt.assert_true(np.allclose(res[0][1].data, np.array(
+                [[  2.37122140e-06,   2.37122140e-06],
+                [  2.37122140e-06,   2.37122140e-06]]), atol=1e-3))
+
+    def test_quant_cross_section(self):
+        s =self.signal
+        method = 'cross_section'
+        factors = [3, 5]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors)
+        nt.assert_true(np.allclose(res[1][0].data, np.array(
+            [[ 16.02176621,  16.02176621],
+            [ 16.02176621,  16.02176621]])), atol=1e-3)
```

Comment:
It may also be cleaner to assign the numpy array containing the expected result outside of the test, but not a big deal there.